src/app/item-popup/WeaponDeepsightInfo.tsx

```diff
@@ -9,18 +14,41 @@ export function WeaponDeepsightInfo({ item }: { item: DimItem }) {
   const record = item.patternUnlockRecord;
   const relevantObjectives = record?.objectives.filter((o) => !o.complete);
 
+  const defs = useD2Definitions()!;
+  const harmonizerIcon = defs.InventoryItem.get(DEEPSIGHT_HARMONIZER).displayProperties.icon;
+
   if (!relevantObjectives?.length) {
     return null;
   }
 
+  const harmonizable = isHarmonizable(item);
+  const harmonizerTooltipText = item.tooltipNotifications?.map((t) => t.displayString);
+  const harmonizableTooltip = (
```

Comment:
Might be nice to pull this out into a component, both for readability and so we don't do all the work for it when the item isn't harmonizable.

Reply:
I've taken a stab at component-ising this, let me know if theres a smarter way to have done so